packages/serverpod/lib/src/relic/static/static_handler.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:serverpod/serverpod.dart';
```

Comment:
Ok, for now IMO, but this may need to be reworked with the new routing update.

Reply:
Fixed!